### PR TITLE
Added Lookups UI Support

### PR DESCRIPTION
**Pull request by sonamgupta21**
# Description
Added UI Support to Upload `.csv` and `.csv.gz` lookup files and to see all the uploaded lookup file names.

image
image

Fixes #<issue-number> (link all the GitHub issues this addresses)

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ ] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: pkg/lookups/lookups.go [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+	if !strings.HasSuffix(strings.ToLower(fileName), allowedExtCSV) &&
+		!strings.HasSuffix(strings.ToLower(fileName), allowedExtCSVGZ) {
+		if isCSVGZ {
+			fileName += allowedExtCSVGZ
+		} else {
+			fileName += allowedExtCSV
+		}
+	}
+
+	fullLookupsDir := filepath.Join(config.GetDataPath(), lookupsDir)
```
Comment (AndrewHess) [inline pkg/lookups/lookups.go:74]: There's already a function for this:
```suggestion
	fullLookupsDir := config.GetLookupPath()
```

And then I think you can delete the `lookupsDir` global constant

File: pkg/lookups/lookups.go [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+		ctx.Error("Error creating lookups directory", fasthttp.StatusInternalServerError)
+		return
+	}
+
+	dstPath := filepath.Join(fullLookupsDir, fileName)
+	if _, err := os.Stat(dstPath); !os.IsNotExist(err) {
+		log.Errorf("UploadLookupFile: File already exists: %s", fileName)
+		ctx.Error("A file with the same name already exists", fasthttp.StatusConflict)
+		return
+	}
```
Comment (AndrewHess) [inline pkg/lookups/lookups.go:86]: Do we want to throw an error here always, or provide a way to write over the existing file? If we can't write over a file, they'd have to manually delete the file and then upload the new version which seems a little annoying. But if we do want to add a way to overwrite a file, I think we could do that in a separate small PR if you want

**Review by AndrewHess — COMMENTED** [review]
It would be good to add the new APIs to the API_README.md file

I didn't check the frontend code

**Review by sunitakawane — APPROVED** [review]

File: pkg/lookups/lookups.go (hunk 2 above)
Comment (sonamgupta21) [reply pkg/lookups/lookups.go:85]: I'll address this in a new PR.

**Pull request merged**